scripts/make_source_dist.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash -ex
+# vim: ts=4 sw=4 expandtab
+ceph_version_git=$(git describe --abbrev=8 --match 'v*' | sed s/^v//)
+
+if command -v podman; then
+  PODMAN=podman
+elif [[ "$(groups)" =~ .*\ docker\ .* ]]; then
+  PODMAN=docker
+else
+  PODMAN="sudo docker"
+fi
+
+printf "FROM ubuntu:24.04\nRUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y dpkg-dev devscripts && apt-get clean && rm -rf /var/lib/apt/lists/*" | $PODMAN build -t ubuntu_builder -
+$PODMAN run --rm -v $PWD:/ceph:z -e DEBEMAIL=contact@ceph.com ubuntu_builder:latest bash -c "cd /ceph && dch -v $ceph_version_git-1 'autobuilder' < /dev/null"
+
+rm -rf dist release
+mkdir -p release
+echo $ceph_version_git > release/version
```

Comment:
Seems like we lost some things: submodule update, 'dirty repo' check, maybe more; was that intentional?

Reply:
ah, submodule update is done in make-dist itself, I see now